source/draw/gpu/GpuSurface.ooc

```diff
@@ -82,9 +82,15 @@ GpuSurface: abstract class extends Canvas {
 	}
 	draw: virtual func ~mesh (image: GpuImage, mesh: GpuMesh) { Debug raise("draw~mesh unimplemented!") }
 	readPixels: virtual func -> ByteBuffer { raise("readPixels unimplemented!"); null }
-	_createTextureTransform: static func (imageSize: IntVector2D, box: IntBox2D) -> FloatTransform3D {
-		scaling := FloatTransform3D createScaling(box size x as Float / imageSize x, box size y as Float / imageSize y, 1.0f)
-		translation := FloatTransform3D createTranslation(box leftTop x as Float / imageSize x, box leftTop y as Float / imageSize y, 0.0f)
+	_createTextureTransform: static func ~Int (imageSize: IntVector2D, box: IntBox2D) -> FloatTransform3D {
+		This _createTextureTransform(imageSize toFloatVector2D(), box toFloatBox2D())
+	}
+	_createTextureTransform: static func ~Float (imageSize: FloatVector2D, box: FloatBox2D) -> FloatTransform3D {
```

Comment:
Maybe this and the function above could also have a `~local` part in the suffix, as `static func ~Normalized` below.